grype/db/v6/installation/curator.go

```diff
@@ -478,12 +483,23 @@ func newMonitor() monitor {
 
 	return monitor{
 		AtomicStage:      stage,
-		downloadProgress: downloadProgress,
-		importProgress:   importProgress,
+		downloadProgress: dummyMonitor{downloadProgress},
+		importProgress:   dummyMonitor{importProgress},
+		hydrateProgress:  dummyMonitor{hydrateProgress},
 	}
 }
 
 func (m monitor) SetCompleted() {
 	m.downloadProgress.SetCompleted()
 	m.importProgress.SetCompleted()
+	m.hydrateProgress.SetCompleted()
+}
+
+type dummyMonitor struct {
```

Comment:
I should probably rename this -- this changes the behavior so that the progress value will be set to the total size when set to completed. This fixes the current progress bar behavior (which today does not move at all when importing).